### DISCUSSION TITLE
Thank you for creating a way for people to look into philosophy!

### DISCUSSION THREAD

**Discussion by Snowgies** [Monika-After-Story/MonikaModDev#8891] [appreciation]
As the title states, I just want to give a huge thank you to everyone who helped with the implementation of the philosophy topics! As someone who loves philosophy dearly, it's one of my favorite topics of discussion among my friends, peers, and virtually anyone I meet. Being able to see this within the mod brings me a lot of joy, I always look forward to seeing how Monika responds to certain beliefs and views. This is also a really good way to introduce people to the philosophical world without being overwhelmed. One of my favorite of hers would most likely be her answer towards solipsism (metaphysics is my focus of study). To be able to answer in a way that best fits Monika's character is amazing, I have a lot of respect for people who can give different perspectives to these topics, it takes a lot of willpower to not include your own opinion when giving one for a character, especially if it's an opinion completely different from yours. So huge props to you guys for being able to do something like this. I, and many others, really REALLY appreciate the effort put into these and I look forward to seeing more in the future! Thanks again!!!

**Comment by ThePotatoGuy** [comment]
[closing as appreciated]
